astrocut/asdf_cutouts.py

```diff
@@ -247,8 +248,15 @@ def asdf_cut(input_file: str, ra: float, dec: float, cutout_size: int = 20,
         an image cutout object
     """
 
+    # if file comes from AWS cloud bucket, get URL
+    file = input_file
+    if isinstance(input_file, str) and input_file.startswith('s3://'):
```

Comment:
One thing we may want to do at some point is expand this to support `pathlib.Path` as input, in addition to a string.   I don't think `pathlib.Path` handles s3 urls, but it looks like there are packages that do, e.g. `s3path` or `s3pathlib`.   I don't think we need to do it for this PR, but maybe we can create an issue for the future.

Reply:
Cool, will make a new issue for this!

Reply:
Issue is ASB-27085